src/Raven.Server/Documents/Sharding/Operations/MultiOperation.cs

```diff
@@ -3,33 +3,39 @@
 using System.Diagnostics;
 using System.Threading;
 using System.Threading.Tasks;
-using Raven.Client.Documents.Commands;
 using Raven.Client.Documents.Operations;
 using Raven.Client.Extensions;
+using Raven.Client.Http;
 
 namespace Raven.Server.Documents.Sharding.Operations;
 
-public class MultiOperation
+public abstract class MultiOperation
```

Comment:
this can be used only in a sharded context, right?
I suggest to name this `ShardedMultiOperation`, `ShardedDatabaseMultiOperation` & `ShardedServerMultiOperation`

Reply:
done